mesheryctl/pkg/utils/helpers.go

```diff
@@ -257,7 +263,30 @@ func PreReqCheck(subcommand string, focusedContext string) error {
 			}
 		}
 	} else if currCtx.Platform == "kubernetes" {
+		if err != nil {
+			return errors.Wrap(err, "failed to create new client")
+		}
```

Comment:
Where are we creating this client? - Where are we catching this error?

Reply:
Sorry, this was a typo. :(